### DISCUSSION TITLE
能否禁止未登录状态下每次启动时都弹出的登录窗口？

### DISCUSSION THREAD

**Discussion by jcfun** [SPlayer-Dev/SPlayer#271] [想法 Ideas]
如题，现在风控比较严重，我的账号总是被踢下线。软件即使不登录也可以使用，因此可不可以不要在每次启动时都弹出登录窗口？

**Comment by imsyy** [comment]
已优化弹窗出现次数，并且在第一次过期并关掉弹窗后，将不会再弹出
